gammapy/background/tests/test_energy_offset_array.py

```diff
@@ -0,0 +1,28 @@
+from ..energy_offset_array import EnergyOffsetArray
+from gammapy.data import DataStore
+from gammapy.data import EventList
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_dependency, requires_data
+import numpy as np
+#import pylab as pt
+
+
+@requires_dependency('matplotlib')
+@requires_data('gammapy-extra')
+def test_energy_offset_array():
+    dir=str(gammapy_extra.dir)+'/datasets/hess-crab4'
+    data_store = DataStore.from_dir(dir)
+    Observation_Table=data_store.obs_table
+    event_list_files = data_store.make_table_of_files(Observation_Table,
+                                                          filetypes=['events'])
+    ev_list=[]
+    for (i,i_ev_file) in enumerate(event_list_files['filename']):
```

Comment:
Please format code following the PEP8 style guide.

If you're using PyCharm, there's a command to do it: https://www.jetbrains.com/pycharm/help/reformatting-source-code.html
This is what I do ... I hit this shortcut after doing some coding, before saving the file.

If you don't have an editor that makes this simple, the [pep8](https://pypi.python.org/pypi/pep8) and [autopep8](https://pypi.python.org/pypi/autopep8) command line tools can help.

In this case:
```
pep8 gammapy/background/tests/test_energy_offset_array.py
```